### PR TITLE
RANCHER-1276 Cypress test fails during nightly run

### PR DESCRIPTION
**Pull request by OHaimanov**
Jira: https://folio-org.atlassian.net/browse/RANCHER-1276

### CODE REVIEW

**Review by epam-avramenko — DISMISSED** [review]

**Review by aatoyan — APPROVED** [review]

**Review by eldiiar-duishenaliev — APPROVED** [review]

**Review by ostapwd — CHANGES_REQUESTED** [review]
Current implementation causes tests to fail

File: pipelines/folioRancher/folioScheduledTesting/runNightlyCypressTests/Jenkinsfile [hunk 1]
```diff
@@ -2,7 +2,8 @@
 import org.folio.models.RancherNamespace
 import org.jenkinsci.plugins.workflow.libs.Library
 
-@Library('pipelines-shared-library@RANCHER-741-Jenkins-Enhancements') _
+@Library('pipelines-shared-library@RANCHER-1276') _
```
Comment (OHaimanov) [inline pipelines/folioRancher/folioScheduledTesting/runNightlyCypressTests/Jenkinsfile:5]: Please change branch before merge

File: pipelines/folioRancher/folioTestingTools/runCypressTests/Jenkinsfile [hunk 2]
```diff
@@ -3,7 +3,8 @@ import hudson.util.Secret
 import org.jenkinsci.plugins.workflow.libs.Library
 import java.util.Calendar
 
-@Library('pipelines-shared-library@RANCHER-741-Jenkins-Enhancements') _
+@Library('pipelines-shared-library@RANCHER-1276') _
```
Comment (OHaimanov) [inline pipelines/folioRancher/folioTestingTools/runCypressTests/Jenkinsfile:6]: Please change branch before merge

File: pipelines/folioRancher/folioScheduledTesting/runNightlyCypressTests/Jenkinsfile (hunk 1 above)
Comment (epam-avramenko) [reply pipelines/folioRancher/folioScheduledTesting/runNightlyCypressTests/Jenkinsfile:5]: Already have done in https://github.com/folio-org/pipelines-shared-library/pull/519

File: pipelines/folioRancher/folioTestingTools/runCypressTests/Jenkinsfile (hunk 2 above)
Comment (epam-avramenko) [reply pipelines/folioRancher/folioTestingTools/runCypressTests/Jenkinsfile:6]: Already have done in https://github.com/folio-org/pipelines-shared-library/pull/519

**Review by epam-avramenko — APPROVED** [review]

**Review by ostapwd — APPROVED** [review]

**Review by epam-avramenko — APPROVED** [review]

**Pull request merged**